accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/rest/controllers/BasicAccessionHistoryController.java

```diff
@@ -44,4 +48,13 @@ public BasicAccessionHistoryController(HistoryService<MODEL, ACCESSION> service,
         return converter.convert(service.getHistory(accession));
     }
 
+    @RequestMapping(value = "/getAccessionMergedInto/{accession}", method = RequestMethod.GET,
+            produces = "application/json")
+    public ResponseEntity<AccessionWrapper<MODEL, String, ACCESSION>> getAccessionMergedInto(@PathVariable ACCESSION
```

Comment:
I don't think this is the functionality we agreed upon. We decided that the regular endpoint to query accessioned objects would only support one accession per request, instead of multiple ones. That way, if it has been merged we can do the redirection without any problems. But this implementation adds a new endpoint and still allows to query multiple accessions in the other one.